modules/ts/src/ts_perf.cpp

```diff
@@ -353,6 +353,53 @@ void Regression::write(cv::Mat m)
     write() << "val" << getElem(m, y, x, cn) << "}";
 }
 
+#define OVER_EPS(name,eps) (eps < fabs(actual_##name - expect_##name))
+void Regression::verifyLoose(cv::FileNode node, cv::Mat actual, double eps, std::string argname, ERROR_TYPE err)
+{
+    if (!actual.empty() && actual.dims < 2) return;
```

Comment:
Perhaps these changes are not needed too.

Reply:
No more modification left in the ts module